.github/workflows/code-quality.yml

```diff
@@ -0,0 +1,43 @@
+name: Code Quality
+
+on:
+  push:
+    branches: [ master ]
+  pull_request: ~
+
+env:
```

Comment:
Please remove this section, this not make sense to keep there, I did wrong on `util` project, I fixed.

Reply:
Done.